WordPress/gutenberg
# ResizableFrame: Account for window resizing

Fixes #52582
Follow-up to #49910

## What?

Make the resizable Editor Canvas to work properly after a browser window resize.

## Why?

The original code only took reference measurements on initial render, so any calculations would be wrong after a browser window resize.

## How?

Take reference measurements from a resize observer instead.

## Testing Instructions

1. Open the [Site Editor](http://localhost:8888/wp-admin/site-editor.php).
2. Resize the editor canvas using your pointing device.
3. Resize the browser window.
4. Resizing the editor canvas should still work the same as in Step 2.

### Testing Instructions for Keyboard


## Screenshots or screencast 


https://github.com/WordPress/gutenberg/assets/555336/39638c73-92ce-4042-bc00-09843a6ab32b
